src/authentication/jwtAuthProvider.tsx

```diff
@@ -32,7 +32,7 @@ export default class JWTAuthProvider extends BaseAuthProvider {
   }
 
   public refresh(): Promise<void> {
-    return Axios.post(`${this.authUrl}/api/jwt/refresh`, {
+    return Axios.post(`${this.authUrl}/refresh`, {
       token: this.token,
     })
       .then((res) => {
```

Comment:
I believe the line after this should be
`this.storeToken(res.data);`
based on the usage of operationsgateway_api and ldap-jwt-auth, the `.token` was removed on the above lines just not here.